cmd/ebrelayer/txs/.env

```diff
@@ -0,0 +1,3 @@
+# Replace sample private key for an active validator address
+# For local testing, use accounts[1] private key from truffle
```

Comment:
i'd prefer we didnt commit `.env` file into the repo so we can config things locally for iso dev/testing

Reply:
OK, I'll fix this.

Reply:
done.